### PR TITLE
Support wildcard category name in export messages config #10359

### PR DESCRIPTION
**Pull request by rmrevin**
Issue #10359

### CODE REVIEW

File: framework/console/controllers/MessageController.php [hunk 1]
```diff
@@ -521,6 +521,31 @@ private function extractMessagesFromTokens(array $tokens, array $translatorToken
     }
 
     /**
+     * @param string $category category that is checked
```
Comment (SilverFire) [inline framework/console/controllers/MessageController.php]: Need a small explanation about how the ways how the method detects, whether the category is ignored

File: framework/console/controllers/MessageController.php [hunk 2]
```diff
@@ -521,6 +521,31 @@ private function extractMessagesFromTokens(array $tokens, array $translatorToken
...
+     * @param array $ignoreCategories message categories to ignore.
+     * @return bool
```
Comment (SilverFire) [inline framework/console/controllers/MessageController.php]: `bool` -> `boolean`

File: framework/console/controllers/MessageController.php [hunk 3]
```diff
@@ -521,6 +521,31 @@ private function extractMessagesFromTokens(array $tokens, array $translatorToken
...
+     */
+    protected function isIgnoredCategory($category, array $ignoreCategories)
```
Comment (SilverFire) [inline framework/console/controllers/MessageController.php]: I'm considering about renaming the method to `isCategoryIgnored`

File: framework/console/controllers/MessageController.php [hunk 4]
```diff
@@ -130,7 +130,7 @@ class MessageController extends Controller
      */
     public $catalog = 'messages';
     /**
-     * @var array message categories to ignore. For example, 'yii'.
+     * @var array message categories to ignore. For example, 'yii', 'app*', 'widgets/menu', etc.
```
Comment (SilverFire) [inline framework/console/controllers/MessageController.php]: It will be good to add a link on the method with `@see` tag

**Comment by SilverFire** [comment]
We need a CHANGELOG line

File: framework/console/controllers/MessageController.php (hunk 1 above)
Comment (rmrevin) [inline framework/console/controllers/MessageController.php]: I don't quite understand what do.

**Comment by rmrevin** [comment]
In CHANGELOG write version 2.0.8?

File: framework/console/controllers/MessageController.php (hunk 1 above)
Comment (SilverFire) [inline framework/console/controllers/MessageController.php]: Sorry there was a spelling mess :) Updated.

I'm talking about examples at most.

Comment (rmrevin) [inline framework/console/controllers/MessageController.php]: My English is poor. Could you help me what to write?

**Comment by SilverFire** [comment]
No, leave it in 2.0.7

**Comment by rmrevin** [comment]
Done.

**Comment by samdark** [comment]
See no issue in the code. Could be merged.

**Comment by SilverFire** [comment]
Thank you!

**Pull request merged**